components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -35,7 +35,6 @@
 static const CGFloat MDCDialogActionsVerticalPadding = 8.0;
```

Comment:
Should this just be updated to 12 to fix the touch area and the overlapping issue?

Reply:
Let's roll-back and we can verify the increased padding solves the issue.
In general, the philosophy is roll-back immediately, then fix.

Reply:
Let's rollback so we can verify the fix.  We definitely don't want this to go out in the release.